docs/api/browser-window.md

```diff
@@ -66,11 +66,14 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
 * `darkTheme` Boolean - Forces using dark theme for the window, only works on
   some GTK+3 desktop environments.
 * `transparent` Boolean - Makes the window [transparent](frameless-window.md).
-* `type` String - Specifies the type of the window, possible types are
-  `desktop`, `dock`, `toolbar`, `splash`, `notification`. This only works on
-  Linux.
-* `standardWindow` Boolean - Uses the OS X's standard window instead of the
-  textured window. Defaults to `true`.
+* `type` String - Specifies the type of the window, which applies
+  additional platform-specific properties.
+  - On Linux, possible types are `desktop`, `dock`, `toolbar`, `splash`, `notification`.
+  - On Mac OS X:
+    - `textured`: Adds metal gradient appearance (NSTexturedBackgroundWindowMask)
```

Comment:
Our docs site can not deal with nested list with >= 3 levels, can you put them into one line? We can probably add a new section in docs if it is too long.

Also can you use `*` instead of `-` to match other places in the docs?